src/figuresClasses.ts

```diff
@@ -2,10 +1,88 @@
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.color = color;
+
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Сторони мають бути більші за 0');
+    }
+
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error(`Sides ${a}, ${b}, and ${c} can't form a triangle.`);
+    }
+  }
+
+  getArea(): number {
+    const p = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(p * (p - this.a) * (p - this.b) * (p - this.c));
+
+    return Math.floor(area * 100) / 100;
+  }
+}
+
+export class Circle implements Figure {
+  shape: 'circle' = 'circle';
 
-export class Triangle implements Figure {}
+  color: 'red' | 'green' | 'blue';
 
-export class Circle implements Figure {}
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    public radius: number,
+  ) {
+    this.color = color;
 
-export class Rectangle implements Figure {}
+    if (radius <= 0) {
+      throw new Error('Радіус має бути більший за 0');
+    }
+  }
+
+  getArea(): number {
+    const radius = Math.pow(this.radius, 2);
+    const pi = Math.PI;
+    const area = radius * pi;
+
+    return Math.floor(area * 100) / 100;
+  }
+}
+
+export class Rectangle implements Figure {
+  shape: 'rectangle' = 'rectangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    public width: number,
+    public height: number,
+  ) {
+    this.color = color;
+
+    if (width <= 0 || height <= 0) {
+      throw new Error('Ширини та висота не можуть бути менше 0 або 0');
```

Comment:
The error message should be in English as per the task requirements. Consider changing it to 'Width and height must be greater than 0'.